File: send.py
import os
import requests
import smtplib
from email.message import EmailMessage

# ---------- DISCORD ----------
def send_discord():
    webhook = os.environ["DISCORD_WEBHOOK"]

    payload = {
        "content": "🎉 **HAPPY BIRTHDAY BRO!** 🎂\nSurprise 👉 https://yourusername.github.io/birthday-site"
    }

    r = requests.post(webhook, json=payload)
    r.raise_for_status()

# ---------- EMAIL ----------
def send_email():
    msg = EmailMessage()
    msg["From"] = os.environ["EMAIL_USER"]
    msg["To"] = os.environ["TO_EMAIL"]
    msg["Subject"] = "🎉 HAPPY BIRTHDAY 🎉"
    msg.set_content(
        "Happy Birthday bro 🎂🔥\n\nSurprise site:\nhttps://yourusername.github.io/birthday-site"
    )

    with smtplib.SMTP_SSL("smtp.gmail.com", 465) as server:
        server.login(os.environ["EMAIL_USER"], os.environ["EMAIL_PASS"])
        server.send_message(msg)

if __name__ == "__main__":
    send_discord()
    send_email()
    print("Birthday wishes sent 🚀")

File: send.js
import https from "https";
import nodemailer from "nodemailer";

// ---------- DISCORD ----------
function sendDiscord() {
  const data = JSON.stringify({
    content: "🎉 **HAPPY BIRTHDAY BRO!** 🎂\nHere’s your surprise 👉 https://yourusername.github.io/birthday-site"
  });

  const url = new URL(process.env.DISCORD_WEBHOOK);

  const req = https.request({
    hostname: url.hostname,
    path: url.pathname,
    method: "POST",
    headers: {
      "Content-Type": "application/json",
      "Content-Length": data.length
    }
  });

  req.write(data);
  req.end();
}

// ---------- EMAIL ----------
async function sendEmail() {
  const transporter = nodemailer.createTransport({
    service: "gmail",
    auth: {
      user: process.env.EMAIL_USER,
      pass: process.env.EMAIL_PASS
    }
  });

  await transporter.sendMail({
    from: `"Birthday Bot 🎂" <${process.env.EMAIL_USER}>`,
    to: process.env.TO_EMAIL,
    subject: "🎉 HAPPY BIRTHDAY 🎉",
    text: "Happy Birthday bro 🎂🔥\nSurprise site:\nhttps://yourusername.github.io/birthday-site"
  });
}

(async () => {
  await sendDiscord();
  await sendEmail();
  console.log("Birthday wishes sent 🚀");
})();
